packages/button/src/button.vue

```diff
@@ -94,7 +94,7 @@ export default defineComponent({
 
   emits: ['click'],
 
-  setup(props, ctx) {
+  setup(props: IButtonProps, { emit }) {
```

Comment:
Actually you don't need to put type annotations here since we use `defineComponent to define the component which will infer the type themselves thanks to `Vue`'s type annotation work. But this is not wrong since it's just type annotation.